385/Assets/Scripts/RopeSystem.cs

```diff
@@ -112,6 +112,13 @@ public class RopeSystem : MonoBehaviour
 
     void Start()
     {
+		// Set axis strings based on the user's controller selection from the menu screen
+		if (ChangeScene.SelectedControllerMode) 
+		{
+			FireAxis = "Fire_P2";

```

Comment:
We should have publicly accessible strings that can be changed in the unity editor at runtime, not hard coded strings.